content/css/sel4.css

```diff
@@ -286,3 +286,61 @@ a.divLink {
   font-size:20px;
   font-weight: bold;
 }
+
+.news {
+  width: auto;
+  height:auto;
+  padding-top:5px;
+  padding-left:10px;
+  padding-right:5px;
+  margin-top:20px;
+  margin-left:60px;
+  margin-right:60px;
```

Comment:
It might make sense to do the padding and margin in `ex` or `em` for this one so that they scale with font size (esp for mobile/tablet).

Reply:
Actually, left/right probably better in pixels, to keep it in sync with the buttons underneath. But top/bottom could be relative to text size.

Reply:
got it. changed top/bottom